shared/teams/channel/index.tsx

```diff
@@ -234,26 +233,14 @@ const Channel = (props: OwnProps) => {
   const renderSectionHeader = ({section}: {section: {title?: string}}) =>
     section.title ? <Kb.SectionDivider label={section.title} /> : null
 
-  // Animation
```

Comment:
these animations will take a bunch of work to get going since the headers are now totally sep from this component. Turns out with the update to the nav this was already broken. I argue this isn't worth supporting at all